src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishASingleFileApp.cs

```diff
@@ -23,13 +23,17 @@ public class GivenThatWeWantToPublishASingleFileApp : SdkTest
 
         private const string PublishSingleFile = "/p:PublishSingleFile=true";
         private const string FrameworkDependent = "/p:SelfContained=false";
-        private const string IncludePdb = "/p:IncludeSymbolsInSingleFile=true";
+        private const string PlaceStamp = "/p:PlaceStamp=true";
         private const string ExcludeNewest = "/p:ExcludeNewest=true";
         private const string ExcludeAlways = "/p:ExcludeAlways=true";
         private const string DontUseAppHost = "/p:UseAppHost=false";
         private const string ReadyToRun = "/p:PublishReadyToRun=true";
         private const string ReadyToRunWithSymbols = "/p:PublishReadyToRunEmitSymbols=true";
         private const string UseAppHost = "/p:UseAppHost=true";
+        private const string IncludePdb = "/p:IncludeSymbolsInSingleFile=true";
+        private const string IncludeNative = "/p:IncludeNativeLibrariesInSingleFile=true";
+        private const string IncludeAllContent = "/p:IncludeAllContentInSingleFile=true";
+        private const string ExcludeContent = "/p:IncludeAllContentInSingleFile=false";
```

Comment:
The `ExcludeContent` flag is only needed until Runtime changes to process assemblies from bundle is available in the installer repo, so that `global.json` can be updated to use the new runtime. Once this is done, the default will be set to `IncludeAllContentInSingleFile=false` in the publish-targets.